modules/post/windows/gather/enum_av_excluded.rb

```diff
@@ -0,0 +1,148 @@
+        OptBool.new('ESSENTIALS', [true, 'Enumerate exclusions for Microsoft Security Essentials', true]),
+        OptBool.new('SEP', [true, 'Enumerate exclusions for Symantec Endpoint Protection (SEP)', true])
+      ]
+    )
+  end
+
+  def enum_mssec
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Microsoft Antimalware")
+      print_status "MS Security Essentials Identified"
+      return true
+    else
+      return false
+    end
+  rescue
+    return false
+  end
+
+  def enum_defender
+    if registry_enumkeys("HKLM\\SOFTWARE\\Microsoft").include?("Windows Defender")
+      print_status "Windows Defender Identified"
+      return true
+    else
+      return false
+    end
+  rescue
+    return false
+  end
+
+  def enum_sep
+    if registry_enumkeys("HKLM\\SOFTWARE\\Symantec").include?("Symantec Endpoint Protection")
+      print_status "SEP Identified"
+      return true
+    else
+      return false
+    end
+  rescue
+    return false
+  end
+
+  def excluded_sep
+    print_status "Excluded Locations:"
+    keyadm = "HKLM\\SOFTWARE\\Symantec\\Symantec Endpoint Protection\\AV\\Exclusions\\ScanningEngines\\Directory\\Admin"
+    if (found_keysadm = registry_enumkeys(keyadm))
+      found_keysadm.each do |vals|
+        full = keyadm + "\\" + vals
+        values = registry_getvaldata(full, "DirectoryName")
+        print_good "#{values}"
+      end
+    else
+      print_error "No Admin Locations Found"
+    end
+
+    keycli = "HKLM\\SOFTWARE\\Symantec\\Symantec Endpoint Protection\\AV\\Exclusions\\ScanningEngines\\Directory\\Client"
+    if (found_keyscli = registry_enumkeys(keycli))
+      found_keyscli.each do |vals|
+        full = keycli + "\\" + vals
+        values = registry_getvaldata(full, "DirectoryName")
+        print_good "#{values}"
+      end
+    else
+      print_error "No Client Locations Found"
+    end
+  end
+
+  def excluded_mssec
+    print_status "Excluded Locations:"
+    keyms = "HKLM\\SOFTWARE\\Microsoft\\Microsoft Antimalware\\Exclusions\\Paths\\"
+    if (found = registry_enumvals(keyms))
+      found.each do |num|
+        print_good "#{num}"
+      end
+    else
+      print_error "No Excluded Locations Found"
+    end
+  end
+
+  def excluded_defender
+    print_status "Excluded Locations:"
+    keyms = "HKLM\\SOFTWARE\\Microsoft\\Windows Defender\\Exclusions\\Paths\\"
+    if (found = registry_enumvals(keyms))
+      found.each do |num|
+        print_good "#{num}"
+      end
+    else
+      print_error "No Excluded Locations Found"
+    end
+  end
+
+  def setup
+    unless datastore['DEFENDER'] || datastore['ESSENTIALS'] || datastore['SEP']
+      fail_with(Failure::BadConfig, 'Must set one or more of DEFENDER, ESSENTIALS or SEP to true')
+    end
+  end
+
+  def run
+    if sysinfo['Architecture'] =~ /WOW64/
+      print_error "You are running this module from a 32-bit process on a 64-bit machine. Migrate to a 64-bit process and try again"
```

Comment:
Fixed in https://github.com/jhart-r7/metasploit-framework/commit/f6417df9ba6c687a3e5d0120b705b3b7cd523b11.